resources/js/components/ReactionListComponent.vue

```diff
@@ -29,13 +27,13 @@
             </a>
 
             <ul :id='reactionsId' class='dropdown-content'>
-                <li><a href="#!" @click="create('thumb_up')">
-                    <i class="material-icons">thumb_up</i>
-                </a></li>
-
-                <li><a href="#!" @click="create('thumb_down')">
-                    <i class="material-icons">thumb_down</i>
-                </a></li>
+                <li 
+                    v-for="img in allReactionsIcons"
+                    :key="img">
```

Comment:
eu colocaria todas essas informações do `v-for` e do `:key` na mesma linha do `<li`. Deixar em linhas separadas, ao meu ver, mais dificulta do que ajuda.